.travis.yml

```diff
@@ -147,9 +147,21 @@ jobs:
     - travis_retry scripts/deploy-to-gcs.sh
     - travis_retry scripts/trigger-build-helm-charts.sh
 
+  - name: Deploy to Demo Environment
+    #if: type in (push, api) AND tag is present
+    install:
+    - travis_retry scripts/setup-tools-for-ubuntu.sh
+    script:
+    # travisqueue.go ensures only a single build runs at a time on Openshift
+    - go run travisqueue.go
+    - travis_retry scripts/setup-openshift.sh ${OC_DEMO_TOKEN}
+    - TILLER_NAMESPACE=console-demo enterprise-suite/scripts/lbc.py install --namespace=console-demo --local-chart=enterprise-suite --set usePersistentVolumes=true,defaultStorageClass=gp2
```

Comment:
`--set usePersistentVolumes=true,defaultStorageClass=gp2` is redundant now.

(Unless this is an update install, which I don't think is the case...)